commands/help-text_test.go

```diff
@@ -0,0 +1,80 @@
+package commands
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+)
+
+//New commands must be added here
+var helpText = map[string]string {
+	"git-lfs-checkout.1.ronn":git_lfs_checkout_HelpText,
```

Comment:
This could just be generated along with the rest of the data. It wouldn't really bother me if it lived at the bottom of `help-text.go`. I think it's too easy to miss adding something to this list. What it's really testing is that the generator has run before building, and if something is missed then the test doesn't have much value.